src/test/javascript/spec/component/text-submission-assessment/text-submission-assessment.component.spec.ts

```diff
@@ -160,25 +171,30 @@ describe('TextSubmissionAssessmentComponent', () => {
         fixture = TestBed.createComponent(TextSubmissionAssessmentComponent);
         component = fixture.componentInstance;
         submissionService = TestBed.inject(SubmissionService);
+        exampleSubmissionService = TestBed.inject(ExampleSubmissionService);
 
         fixture.detectChanges();
     });
 
+    afterEach(() => {
+        sinon.restore();
+    });
+
     it('should create', () => {
-        expect(component).toBeTruthy();
+        expect(component).to.be.ok;
```

Comment:
When looking up the definition I saw that they recommend the explicit value: https://www.chaijs.com/api/bdd/#method_ok
So I assume for this use case it would be `.to.not.be.undefined`.

Other question: What's the reason behind this change?

Reply:
this test suit is very old, we are not using Jasmine or we try to avoid them. As you can see, I imported sinon and chai, and I decided to adapt the whole test suit in [recommended way ](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client-tests/)

Reply:
this test suit is very old, we are not using Jasmine or we try to avoid them. As you can see, I imported sinon and chai, and I decided to adapt the whole test suit in [recommended way ](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client-tests/)

Reply:
Thanks for the explanation makes sense. Then I'll ask the person that wrote the documentation.